addons/l10n_it_edi_withholding/data/template/account.tax-it.csv

```diff
@@ -1,49 +1,49 @@
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","","2630","100",""
-"4ainps","","4% Contributo INPS","4% INPS","5","4.0","percent","purchase","","tax_group_pension_fund","","","","","","","True","TC22","base","invoice","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","invoice","","4402","100",""
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","","4402","100",""
-"4vcp","","4% Contributo Fondo Pensione","4% F.Pens.","20","4.0","percent","sale","","tax_group_pension_fund","","","","","","","True","TC01","base","invoice","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","invoice","","2630","100",""
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","","2630","100",""
-"4acp","","4% Contributo Fondo Pensione","4% F.Pens.","20","4.0","percent","purchase","","tax_group_pension_fund","","","","","","","True","TC01","base","invoice","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","invoice","","4402","100",""
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","","4402","100",""
-"enasarcov","","8.5% Ritenuta ENASARCO","ENASARCO","13","-8.5","percent","sale","","tax_group_enasarco","","","","","on_payment","1611","","TC07","base","invoice","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","invoice","-ENASARCOV","2630","100",""
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","+ENASARCOV","2630","100",""
-"enasarcoa","","8.5% Ritenuta ENASARCO","ENASARCO","13","-8.5","percent","purchase","","tax_group_enasarco","","","","","on_payment","1611","","TC07","base","invoice","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","invoice","+ENASARCOA","2630","100",""
-"","","","","","","","","","","","","","","","","","","base","refund","","","100",""
-"","","","","","","","","","","","","","","","","","","tax","refund","-ENASARCOA","2630","100",""
+"id","description","invoice_label","name","sequence","amount","amount_type","type_tax_use","price_include","tax_group_id","active","tax_scope","l10n_it_withholding_type","l10n_it_withholding_reason","tax_exigibility","cash_basis_transition_account_id","include_base_amount","l10n_it_pension_fund_type","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","repartition_line_ids/use_in_tax_closing","name@it"

```

Comment:
So you are saying that `repartition_line_ids/document_type`, `repartition_line_ids/tag_ids`, `repartition_line_ids/account_id` or `repartition_line_ids/factor_percent` didn't work either?
What's the difference with `repartition_line_ids/use_in_tax_closing`?

Reply:
Added a couple tests here https://github.com/odoo/odoo/pull/157918/commits/f569023db09766aae1cb352402d03963cad4078f that fail before the following commit https://github.com/odoo/odoo/pull/157918/commits/af103266dfd2ce56eff35d8b1ee064219f7ef3ef, but succeed after it is applied.